tfe/resource_tfe_oauth_client.go

```diff
@@ -72,13 +80,19 @@ func resourceTFEOAuthClientCreate(d *schema.ResourceData, meta interface{}) erro
 	// Get the organization and provider.
 	organization := d.Get("organization").(string)
 	serviceProvider := tfe.ServiceProviderType(d.Get("service_provider").(string))
+	privateKey := d.Get("private_key").(string)
```

Comment:
I really like to keep everything ordered nicely so that when thing evolve (get bigger) it remains readable. So it by no means required, but having these kinds of list ordered alphabetically does make me happy :wink:

Reply:
Fixed all the ordering, thanks!